app/__init__.py

```diff
@@ -81,17 +81,25 @@
 authenticated_service = LocalProxy(lambda: g.authenticated_service)
 
 
-def create_app(application):
+def create_app(application, worker_id=None):
```

Comment:
These changes relate to [flask-sqlalchemy 698](https://github.com/pallets-eco/flask-sqlalchemy/issues/698), which was fixed in flask-sqlalchemy v3.0.0 ([changelog](https://flask-sqlalchemy.palletsprojects.com/en/3.0.x/changes/#version-3-0-0)).  During a unit test run, this function is called from the "notify_api" fixture in tests/conftest.py.